.github/workflows/linux-eic-shell.yml

```diff
@@ -394,7 +394,13 @@ jobs:
       matrix:
         beam: [10x100, 18x275]
         minq2: [1000]
```

Comment:
This is now redundant.
```suggestion
```

Reply:
But we can keep it if we will want to revert back to the original at some point, but I think testing different geometries is a good idea.